website/docs/advanced/feature-toggle-variants.md

```diff
@@ -61,7 +61,7 @@ In the example above, 60 divides cleanly by three. In cases where the remainder
 #### Overrides
 
 :::note
-Overrides are intended to be used for one-off exceptions during development and may not be suitable for other use cases.
+Overrides are intended to be used for one-off exceptions and during development and may not be suitable for other use cases.
```

Comment:
Exception feels like such a special word, how do you feel about swapping this with "cases" instead?

```suggestion
Overrides are intended to be used for one-off cases and during development and may not be suitable for other use cases.
```

Reply:
Love it! ❤️